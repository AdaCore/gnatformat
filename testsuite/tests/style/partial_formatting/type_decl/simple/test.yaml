description: ConcreteTypeDecl formatting for line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 31",
    "-EL 32",
  ]
status_code: 1