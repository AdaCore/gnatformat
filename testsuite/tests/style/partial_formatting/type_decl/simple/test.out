<CompilationUnit test.ads:27:1-37:37>
test.ads(<ConcreteTypeDecl ["T"] test.ads:31:4-31:33>) - 31:1-31:33
^
   type T is tagged null record;$

